packages/rooks/.size-limit.js

```diff
@@ -0,0 +1,42 @@
+// @ts-check
+
+// Regularly update this max size when new hooks are added (keep a threshold)
+const fullBundleMaxSize = "12KB";
```

Comment:
@imbhargav5 just to mention, if new hooks are added and the total size of lib grow, this should be increased.

On the other side, it's totally possible to keep only the `"ESM Webpack ({ useDebounce })"` (see below), that's actually the important check

Reply:
@belgattitude Ah. I think we could write a small script to autopopulate new hooks into the size-limit file when new hooks are added and keep the limit at 2kb. What do you think?

Reply:
Nice idea